### ISSUE TITLE
EPUB Exports: Language isn’t specified (book.opf)

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4443]
For EPUB exports, the language designation is in the metadata, but not declared for package. 

Setting the language ensures that assistive technologies correctly interpret and render the text and that reading systems can make language enhancements available for users.

More information on this in the [Daisy Accessible Publishing Knowledge Base](https://kb.daisy.org/publishing/docs/epub/language.html).

The user reporting this is expecting something like this:

`<package … xml:lang="en">`

**Comment by SteelWagstaff** [comment]
This seems pretty straightfoward and easy to implement. Are we sure the lang attribute is present in the source book but isn't being set correctly in the opf file in the export? Here's what I see in the EPUB generation routine: https://github.com/pressbooks/pressbooks/blob/4ef0ff44e90e41a1968948cfd121cbd5c515c2eb/inc/modules/export/epub/class-epub.php#L2712

I believe we try to set the lang value in the opf file. If it's not happening, we'd need to investigate why and in what situations this is failing, IMO.
